_posts/en/2019-10-21-schnorr-taproot-workshop.md

```diff
@@ -0,0 +1,189 @@
+  or multiple scripts into a tweaked public key. Shows how to spend a segwit v1
+  output using either the key path spend (with a single signature) or a script
+  path spend.
+4. [Case studies](#case-studies) - demonstrates some practical applications
+  of the new schnorr and taproot technologies.
+
+Slides from all the presentations can be downloaded [here][slides].
+Additionally, Bryan Bishop has provided a [transcript][] of the New York
+session.
+
+## Introduction
+
+[![Introduction](/img/posts/taproot-workshop/introduction.png)](https://www.youtube.com/watch?v=1gRCVLgkyAE&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC)
+{:.center}
+
+John Newbery introduces the workshop, giving a summary of why schorr and
+taproot are useful technologies. He also explains why Bitcoin Optech created
+the workshop, and what the aims of the workshop are.
+
+## Preparation and Basic Math
+
+This section shows how to setup the jupyter notebook, gives a refresher on
+basic elliptic curve math and introduces tagged hashes.
+
+#### 0.1 Test Notebook
+
+Before starting the workshops, users should follow the instructions in the
+repository [README][readme], clone the [workshop repository][] and run through
+the test notebook to ensure that their environment is set up correctly.
+
+#### 0.2 Elliptic Curve Math
+
+[![Introduction](/img/posts/taproot-workshop/elliptic-curve-math.png)](https://www.youtube.com/watch?v=oix8ov9iGgk&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=2)
+{:.center}
+
+Elichai Turkel gives a refresher on the basic elliptic curve math that will be
+required for this workshop.
+
+#### 0.3 Tagged Hashes
+
+_(No video)_ This chapter introduces _Tagged hashes_, which are used in both
+the bip-schnorr and bip-taproot proposals.
+
+## Schnorr signatures and MuSig
+
+This section explains the bip-schnorr proposal and explains how MuSig can be
+used to aggregate multiple public keys and partial signatures into a single
+pubkey and signature.
+
+#### 1.1 Schnorr Signatures
+
+[![Schnorr Signatures](/img/posts/taproot-workshop/schnorr.png)](https://www.youtube.com/watch?v=wybiVFdknhg&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=3)
+{:.center}
+
+Elichai explains the mathematics behind schnorr signatures and explains the
+bip-schnorr proposal.
+
+#### 1.2 MuSig
+
+[![MuSig](/img/posts/taproot-workshop/musig.png)](https://www.youtube.com/watch?v=5MbTptrXEC4&list=PLPrDsP88ifOVTEJf_jQGunDUS05M9GdIC&index=4)
+{:.center}
+
+Elichai describes the [MuSig algorithm][musig] (authored by Gregory Maxwell,
+Andrew Poelstra, Yannick Seurin and Pieter Wuille), and shows how it can be
+used to aggregated multiple public keys and partial signatures into a single
+pubkey/signature.
+
+## Taproot
+
+This section explains the bip-taproot and bip-tapscript proposals. It shows the
+format of a segwit v1 output and how such an output can be spent in a key path
+spend or a script path spend. It shows how a tweaked public key can commit to
```

Comment:
"shows" is used repeatedly, consider varying with "demonstrates", "discusses", "runs through", "summarizes", etc.